packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -75,13 +86,20 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
   } = props;
   let {styleProps} = useStyleProps(props);
 
+  let {size, borderWidth, arrowRef} = useArrowSize();
+  let borderDiagonal = borderWidth * ROOT_2;
+  let primary = size + borderDiagonal;
+  let secondary = primary * 2;
+  let arrowTotalCrossPadding = 4;
```

Comment:
We'll need to have a different value for medium vs large scale for positioning consistency. You can access the current scale via `useProvider` Not a blocker though

Medium:
<img width="130" alt="image" src="https://user-images.githubusercontent.com/9661127/222859787-e537cc5b-9bc3-484e-93d1-b4319bb33d68.png">


Large:
<img width="156" alt="image" src="https://user-images.githubusercontent.com/9661127/222859723-71a9e61d-9b17-4850-ae75-3fbe9de86ab9.png">

Reply:
should be equivalent to `--spectrum-popover-border-radius`, it might be a merged variable though, so we can add later, as @LFDanLu said, not a blocker
https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/spectrum-css-temp/components/popover/index.css#L47

Reply:
`--spectrum-popover-border-radius` is not defined (lost in some compilation step?) I used borderRadius directly from the overlay element to make it work